graphql_compiler/compiler/filters.py

```diff
@@ -830,12 +832,18 @@ def _get_filter_op_name_and_values(directive):
         raise AssertionError(u'op_name not found in filter directive arguments!'
                              u'Validation should have caught this: {}'.format(directive))
 
+    if args['op_name'].value.value not in UNARY_FILTERS and 'value' not in args:
+        raise GraphQLValidationError(u'Filter directive value argument omitted for non-unary '
+                                     u'filter operation: {} \nPlease provide a value argument'
```

Comment:
Avoid newline characters in error messages -- they tend to cause issues, for example in logging systems that might assume that newline characters are the separator between different log lines.